ethereumj-core/src/main/java/org/ethereum/listener/RecommendedGasPriceTracker.java

```diff
@@ -38,7 +38,7 @@
  * (if {@link #getPercentileShare()} is not overridden) of the latest transactions were
  * executed at this or lower price.
  */
-public class RecommendedGasPriceTracker extends EthereumListenerAdapter {
+public class RecommendedGasPriceTracker {
```

Comment:
This change breaks backward compatibility. We also need to introduce a mechanism for easy subscribing to the new publisher.

Reply:
Does anyone use this class except us? I thought that this class for our internal logic only.

Reply:
Ok, I'll revert this change.

Reply:
By the way we use this class nowhere =)